src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { useIntl } from 'react-intl';
+
+import AuthWrapper from 'components/AuthWrapper';
+import EmailForm from './EmailForm';
+import PasswordForm from './PasswordForm';
+import TokenForm from './TokenForm';
+
+export const RESET_PASSWORD_STEPS = {
+  initial: 0,
+  emailSent: 1,
+  updatePassword: 2,
+};
+
+const ForgotPassword = () => {
+  const intl = useIntl();
+  const [resetStep, setResetStep] = useState(RESET_PASSWORD_STEPS.initial);
+  const [token, setToken] = useState();
+
+  const handleStepChange = (step) => setResetStep(step);
+
+  const handleSaveToken = (newToken) => setToken(newToken);
+
+  return (
+    <AuthWrapper title={intl.messages['common.resetPassword']}>
+      <>
```

Comment:
this fragment is not necessary, right?

Reply:
yeah. Props validation expects only one element, not an array. If I don't add the fragment it thinks it's an array even when they are conditionally rendered and only one renders at a time